source/Playnite/Emulators/Definitions.yaml

```diff
@@ -1520,3 +1520,105 @@
       Platforms: [Adobe Flash]
       ImageExtensions: [swf]
       ExecutableLookup: ^flashplayer_\d+_sa\.exe$
+
+- Name: bsnes
+  Website: 'https://bsnes.byuu.org/'
+  Profiles:
+    - Name: Default
+      DefaultArguments: '"{ImagePath}"'
+      Platforms: [Super Nintendo Entertainment System]
+      ImageExtensions: [sfc, smc, zip]

```

Comment:
From v108 7z is also supported.

Reply:
Thanks, fixed it.
And you are right, Satellaview is checked now. I wonder if I saw wrong or what happened but it's working now.